src/main/java/models/Player.java

```diff
@@ -0,0 +1,16 @@
+package models;
+
+public class Player {
+
+    private String name;
+    private PlayerType type;
+
+    public Player(String name, PlayerType type){
+        this.name = name;
+        this.type = type;
+    }
+
+    public PlayerType getType() {return type;}

```

Comment:
Maybe use to be more consistent with the constuctor and the getter/setter principle.
```suggestion
    public PlayerType getType() {return this.type;}

```